.gitmodules

```diff
@@ -1,3 +1,15 @@
 [submodule "lib/UOSM-Core"]
 	path = lib/UOSM-Core
 	url = git@github.com:UOSupermileage/UOSM-Core.git
+[submodule "Adafruit_BusIO"]
```

Comment:
I doubt this is needed. You should add the library as a dependency in the PlatformIO.ini.